cpanfile

```diff
@@ -61,4 +61,5 @@ on 'test' => sub {
   requires 'Test::More', '0.88';
   requires 'Test::Warnings';
   requires 'JSON::Validator';
+  requires 'Perl::Tidy', '== 20201001';
```

Comment:
Why is it needed? Not sure, if we should touch cpanfile and change it within PR about baremetal.

Reply:
I removed this - this was here by accident, sorry